src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -120,7 +120,7 @@ class OnfidoStep extends React.Component {
                                 && this.props.walletOnfidoData.loading && <FullscreenLoadingIndicator />}
                             {!this.props.walletOnfidoData.loading && this.props.walletOnfidoData.error && (
                                 <>
-                                    <Text style={[styles.h3, styles.mb2]}>
+                                    <Text style={[styles.h3, styles.textBold, styles.mb2]}>
```

Comment:
There's already a style names `testStrong`

https://github.com/Expensify/Expensify.cash/blob/d3b3a797584054364d96effd00b746bcca77da82/src/styles/styles.js#L96

This has fontFamily that is the reason for `textBold`. Not sure whether we can use this instead of `textBold` (for all use cases)

Reply:
I'm recommending we use the existing class `textStrong` here and remove the new `textBold` class. There are currently no usages of `fontWeight: fontWeightBold,` without `fontFamily: fontFamily.GTA_BOLD,` so I think we should keep it consistent.